src/components/Settings/SoftwareUpgradeSettings.vue

```diff
@@ -95,24 +106,34 @@ const releases = ref<IRelease[]>([]);
 const current = ref<IRelease>();
 const minimum = ref<IRelease>();
 const selectedRelease = ref<string>();
+const showPrereleases = ref<boolean>(false);
+const loadedClientReleases = ref<IRelease[]>([]);
 
 onMounted(async () => {
   const clientReleases = await AppService.getClientReleases();
   current.value = clientReleases.current;
   minimum.value = clientReleases.minimum;
-  releases.value = clientReleases.filter((release) => {
-    const isMinimumVersion = minor(release.tag_name) === minor(minimum.value.tag_name);
-    const isReleaseCandidate = release.prerelease || release.tag_name.includes("rc");
-    const isUnstable = release.draft || release.prerelease;
-    return isMinimumVersion && !isReleaseCandidate && !isUnstable;
-  });
+  loadedClientReleases.value = clientReleases.releases;
+
   const versionSpec = await AppService.getVersion();
   serverVersion.value = versionSpec.version;
   monsterPiVersion.value = versionSpec.monsterPi;
 });
 
+const filteredReleases = computed(() => {
```

Comment:
I've added a computed property to calculate the list on the go, this allowed implementing #325.